okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -491,6 +498,11 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
         + "   \"description\" : \"get modules that provide some interface\" "
         + " }, "
         + " { "
+        + "   \"permissionName\" : \"okapi.import.modules.post\", "
+        + "   \"displayName\" : \"Okapi - import modules\", "
+        + "   \"description\" : \"Import modules\" "
```

Comment:
If we decide that we need a new permission.

Then we should 

* use a permission name the reflects the similarity to okapi.proxy.tenants.post
* copy the wording we use in the raml file.

Example:

    "permissionName" : "okapi.proxy.tenants.post-list"
    "displayName" : "Okapi - announce new modules to proxy"
    "description" : "Announce a list of new modules to the proxy"

Reply:
I'm lazy and will reuse the existing permission.